### PR TITLE
[CI] Update Quarkus versions

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

File: .github/workflows/local_tests.yml [hunk 1]
```diff
@@ -37,33 +37,33 @@ jobs:
...
             mandrel-version: '23.0.1.2-Final'
             java-version: '20'
             os: 'windows-2022'
             timeout-minutes: 80
-          - quarkus-version: '3.2.3.Final'
+          - quarkus-version: '3.2.6.Final'
             mandrel-version: '23.0.1.2-Final'
             java-version: '20'
             os: 'ubuntu-20.04'
             timeout-minutes: 130
```
Comment (zakkak) [inline .github/workflows/local_tests.yml:60]: Does it still make sense to keep JDK 20 tests around?

Comment (jerboaa) [reply .github/workflows/local_tests.yml:60]: With JDK 21 released I'd say no.

**Review by jerboaa — APPROVED** [review]

File: .github/workflows/local_tests.yml (hunk 1 above)
Comment (Karm) [reply .github/workflows/local_tests.yml:60]: We don't have a JDK 21 release, so let's keep them there until we do :rofl:

**Comment by zakkak** [comment]
> Re--running CI ad
> 
> ```
>   72 Terminating due to java.lang.OutOfMemoryError: Java heap space
>   73 The Native Image build process ran out of memory.
>   74 Please make sure your build system has more memory available.
> ```

Hmm, this is the second time we observe this... Are we setting `-J-Xmx5g`?

**Comment by Karm** [comment]
@zakkak We are not IMHO.

**Comment by zakkak** [comment]
> @zakkak We are not IMHO.

Might be worth trying it if issues persist.

**Pull request merged**